src/sentry/api/serializers/models/project.py

```diff
@@ -297,7 +297,8 @@ def get_session_stats(self, project_ids):
         current_interval_start = now - (segments * interval)
         previous_interval_start = now - (2 * segments * interval)
 
-        project_health_data_dict = get_current_and_previous_crash_free_rates(
+        project_health_data_dict = releasehealth.get_current_and_previous_crash_free_rates(
+            org_id=0,  # FIXME(jjbayer): With metrics, this needs an actual org_id
```

Comment:
would make this an optional param and resolve the org ID based on project IDs within the metrics releasehealth backend, and throw if that resolves to more than one org. `Project.objects.get_many_from_cache` should work for this.

At a later point we can refactor this interface such that those extra lookups are not necessary. I think we will later need the org ID for feature flag querying too.